assets/src/scss/editorOverrides.scss

```diff
@@ -76,6 +75,7 @@
   font-size: $font-size-sm;
 }
 
+.wp-block-button .wp-block-button__link[role="textbox"],
 .wp-block-button.is-style-secondary .wp-block-button__link[role="textbox"],
```

Comment:
We can actually probably remove that line 🤔

Reply:
You mean the line with the secondary style? Indeed we could, but since it's only the editor styles it's not that crucial. And it kind of helps the selector still shows it's a secondary style.